logging/tests/unit/test__helpers.py

```diff
@@ -105,15 +105,23 @@ def test_metadata_does_not_exist(self):
         self.assertIsNone(metadata)
 
     def test_request_exception(self):
+        import requests
+
         metadata_key = 'test_url_cannot_connect'
         metadata_url = 'http://metadata.invalid/'
 
-        patch = mock.patch(
+        requests_get_mock = mock.Mock(spec=['__call__'])
```

Comment:
@tseaver I think `spec=()` will produce the same results (i.e. `__call__` isn't actually necessary)